index.html

```diff
@@ -25,15 +25,17 @@
   </head>
   <body>
     <header>
-      <a href="#" class="logo">Idrissa</a>
-
       <nav>
-        <img src="./img/menu.svg" class="img" alt="" />
-        <div class="lists">
+        <a href="./index.html" id="logo">Idrissa</a>
+        <div class="icons">
+          <i class="fa fa-bars display-icon"></i>
+          <i class="fa fa-times no-display-icon"></i>
+        </div>
+        <div class="ul-list">

```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Please, try adjusting your styles so the menu container talked the whole device height. As of right now, a user can choose to ignore your menu list items and just scroll down to your page. Try to adjust to styles to improve the user experience. check out the video below to see your mobile menu in action:

https://user-images.githubusercontent.com/59397159/170011897-08e5f9e6-bdde-4849-842f-cfa95921b25f.mp4

Reply:
**[REQUIRED CHANGES]**

- Please, try reducing the size of your canceled image to match the Figma design

**you design**
![canYours](https://user-images.githubusercontent.com/59397159/170012725-d085a9e2-4333-4650-ad27-4cb68b26375d.png)

**figma design**
![canFigma](https://user-images.githubusercontent.com/59397159/170012730-1793455e-50c3-4e53-97cb-f94c6261d2f5.png)